docs/concepts/trading-on-vega/data-sources.md

```diff
@@ -1,103 +1,113 @@
-
-* Type of data source (signed message, internal Vega market data, date/time, Ethereum, etc.)
-* Data type (e.g. float for a price) - this must be compatible with the usage of the data source (if it is a settlement price, a numeric value would be required; for a trading termination trigger which consumes no data then any data type, etc.). Note that it is possible to have more than one “compatible” type, for instance it might be that a number could be a string or a raw numeric value in a JSON data source.
-* Data source specific details (for signed message, public key of sender; for Ethereum, contract address, method name; etc.)
+* Type of data source (e.g. `vega.builtin.timestamps`, `Open Oracle`)
+* Data type (e.g. float for a price)
+* Data source specific details
 
 Data sources must be able to emit the following data types:
+* Number - for prices or in filter comparisons
+* String - to be used to compare against in filters
+* Date/Time - to compare against in filters
+* Structured data records - such as a set of key value pairs (inputs to filters)
 
-* Number (for prices or in filter comparisons)
-* String (to be used to compare against in filters)
-* Date/Time (to compare against in filters)
-* Structured data records, such as a set of key value pairs (inputs to filters)
+## Signed message data source
+Signed message data sources are a source of off-chain price data, and can be used for settling and terminating a market. They introduce a Vega transaction that represents a data result that is validated by ensuring the signed message is provided by the Ethereum public key provided in the market’s proposal.
 
-Data sources nominated for use by Vega may refer to other data sources. For instance, one that takes a source of structured data records as input and emits only the value of a named field (e.g. to return BTCUSD_PRICE from a record containing many prices) or one that takes another data source as input and emits only data that matches a set of defined filters (e.g. to return only records with specific values in the timestamp and ticket symbol fields).
-
-## Signed message data sources
-Signed message data sources are the first external data source to be supported on Vega, and can be nominated for settling a market. They introduce a Vega transaction that represents a data result. The result is validated by ensuring the signed message is signed by one of a set of public keys provided in a market’s proposal.
-
-:::info
-There are no rewards associated with being a signed message data source, nor are there fees associated with being or using a signed message data source.
-:::
+Specifically, Signed Message Data Sources are equivalent to Posters in [Compound’s Open Price Feed](https://medium.com/compound-finance/announcing-compound-open-oracle-development-cff36f06aad3), taking signed price reports and posting them to the Vega chain. As Open Oracle reports include signatures, the data can still be verified against its source. 
```

Comment:
This is not quite true: the signed message source is the public key and the specification of the fields and filters to process the incoming data, so it is more like the Reporter. Whoever submits the data for the signed message source to Vega is the equivalent of the Poster, but I'm not sure we've named them really.

So in the example the Compound price feed is the signed message data source, the "Reporter" is whoever does the signing, and anyone who submits that txn to Vega is the "Poster"

Reply:
```suggestion
Specifically, the signer of the Signed Message Data Sources is equivalent to the Reporter in [Compound’s Open Price Feed](https://medium.com/compound-finance/announcing-compound-open-oracle-development-cff36f06aad3). As Open Oracle reports include signatures, the data can still be verified against its source. The Poster equivalent is the Vega key that submits the Signed Message to the Vega chain when the market needs it.
```